### DISCUSSION TITLE
R console won't start -- "Kallichore server"  not found

### DISCUSSION THREAD

**Discussion by fabian-s** [posit-dev/positron#5605] [Q&A] [answered]
Error message:

> The terminal process "/usr/share/positron/resources/app/extensions/positron-supervisor/resources/kallichore/kcserver '--port', '62076', '--token', '/tmp/kallichore-d532b7ab.token', '--log-level', 'debug', '--log-file', '/tmp/kallichore-d532b7ab.log'" failed to launch (exit code: 127).

Screenshot:

![image](https://github.com/user-attachments/assets/d1fb2451-d56c-4abf-ba05-a5674dd62e3b)

This is on Linux Mint 20.

**Comment by jmcphers** [comment, marked as answer]
Unfortunately we do not currently support Linux Mint 20 (which is approximately equivalent to Ubuntu 20); this component needs at least Ubuntu 22. You can read more and comment/upvote here:

https://github.com/posit-dev/positron/issues/3854

Although it won't fix your problem, it's also the case that we should show a more informative error here. I've written that up as well:

https://github.com/posit-dev/positron/issues/5611

**Comment by doug-leasure** [comment]
@jmcphers, I have the same issue on Windows 11 [Pro Education v10.0.22631]. 

When positron initially launches the R console, I periodically get the following error: 
```
R 4.4.2 starting.
Kallichore server did not start after 1616ms
```

Shutting down the R console (not restarting) will allow the R console to re-launch successfully.

**Reply by jmcphers** [reply]
What version of Positron are you using? That looks like an error from an older version; could you try with the latest? https://github.com/posit-dev/positron/releases/tag/2025.02.0-171
